### PR TITLE
Feature/tt 587 delete media

### PR DESCRIPTION
**Pull request by sarahentzel**
Delete Media and implement Media from drawer with multiple plans

### CODE REVIEW

File: src/components/MediaTab.tsx [hunk 1]
```diff
@@ -185,11 +199,12 @@ export function MediaTab(props: IProps) {
     passageSections,
     sections,
     queryStore,
+    updateStore,
     auth,
+    projectplans,
   } = props;
   const classes = useStyles();
-  const [keyMap] = useGlobal('keyMap');
```
Comment (gtryus) [inline src/components/MediaTab.tsx]: I think we will want to keep this line.

File: src/components/MediaTab.tsx [hunk 2]
```diff
@@ -286,17 +315,30 @@ export function MediaTab(props: IProps) {
...
     );
-  }, [plan, mediaFiles, passages, passageSections, sections]);
+  }, [projectplans, mediaFiles, passages, passageSections, sections]);
 
   useEffect(() => {
     if (loaded && currentlyLoading + 1 === uploadList.length) {
-      setMessage(<span>Upload complete.</span>);
       // wait to do this to give time for duration calc
       setTimeout(() => {
+        setMessage(<span>Upload complete.</span>);
```
Comment (gtryus) [inline src/components/MediaTab.tsx]: We should probably include this string in the localization file.

File: src/components/ShapingTable.tsx [hunk 3]
```diff
@@ -170,7 +173,7 @@ export default function ShapingTable(props: IProps) {
     }
   };
   const noRow = () => <></>;
-
+  const noCols = () => <span>No Columns Showing</span>;
```
Comment (gtryus) [inline src/components/ShapingTable.tsx]: We probably want to use a localized string here

File: src/components/MediaTab.tsx [hunk 4]
```diff
@@ -28,8 +28,10 @@ import ShapingTable from './ShapingTable';
 import related from '../utils/related';
 import Auth from '../auth/Auth';
 import moment from 'moment';
-import 'moment/locale/fr';
 import { remoteIdNum } from '../utils';
+import { useGlobal } from 'reactn';
+import { keyMap } from '../schema';
```
[the remark this answers is not in the document]
Comment (sarahentzel) [reply src/components/MediaTab.tsx]: This works here.

File: src/components/MediaTab.tsx [hunk 5]
```diff
@@ -28,8 +28,10 @@ import ShapingTable from './ShapingTable';
 import related from '../utils/related';
 import Auth from '../auth/Auth';
 import moment from 'moment';
-import 'moment/locale/fr';
```
[the remark this answers is not in the document]
Comment (sarahentzel) [reply src/components/MediaTab.tsx]: this gives me a warning and seemed not to be needed.  I've put it back.

File: src/components/ShapingTable.tsx (hunk 3 above)
Comment (sarahentzel) [reply src/components/ShapingTable.tsx]: I did think about that,  This won't ever show unless we let the user remove columns but it's required to compile.   I'll add it anyway

File: src/components/MediaTab.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (gtryus) [reply src/components/MediaTab.tsx]: Dates in French are day month year and may be separated by . instead of /

**Pull request merged**